panou/src/App.tsx

```diff
@@ -1,17 +1,28 @@
 import React from "react";
-// import "./App.css";
-// import Registration from "./components/Registration/Registration";
-import Login from "./components/Login/Login";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import AppProvider from "./context/AppProvider";
 // import Modal from "./components/Modal/Modal";
+import PaginaPrincipala from "./components/HomePage/HomePage";
+import Registration from "./components/Registration/Registration";
+import Login from "./components/Login/Login";
+import NavBar from "./components/Navigation/NavBar/NavBar";
+import Container from "./components/Container/Container";
 
 function App() {
   return (
     <div className="App">
-      {/* <Registration /> */}
-      <Login />
-      {/* <Modal>
-        <Login />
-      </Modal> */}
+      <BrowserRouter>
+        <AppProvider>
+          <Container>
+            <NavBar />
+            <Routes>
+              <Route path="/registration" element={<Registration />}></Route>
```

Comment:
aici se poate de facut in modul urmator:

in fisier routers.ts
```export const routes: Route[] = [
{ name: 'Dashboard', path: '/', exact: true, Component: Dashboard },
];
```

iar in ```App.tsx``` sa pui in modul urmator:

```
{routes.map(({ Component, ...item }) => (
              <Route key={item.name} name={item.name} path={item.path} exact={item.exact}>
                <Component />
              </Route>
            ))}```